GravatarApp/ProfileEditor/ProfileEditorView.swift

```diff
@@ -16,15 +15,15 @@ struct ProfileEditorView: View {
                 profile: viewModel.userSession.profile,
                 topPadding: topPadding,
                 imageURL: headerAvatarURL,
-                forceRefresh: $forceRefresh
+                forceRefresh: $forceRefreshAvatar
             )
         } stickyHeader: { opacity, safeArea in
             ProfileEditorStickyHeaderView(
                 profile: viewModel.userSession.profile,
                 opacity: opacity,
                 safeAreaInsets: safeArea,
                 imageURL: headerAvatarURL,
-                forceRefresh: $forceRefresh
+                forceRefresh: $forceRefreshAvatar
```

Comment:
The profile editor didn't share the same force refresh flag so that was one problem.